paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/DefaultEventReporter.kt

```diff
@@ -36,13 +36,11 @@ internal class DefaultEventReporter @Inject internal constructor(
 ) : EventReporter {
 
     private var isDeferred: Boolean = false
+    private var linkEnabled: Boolean = false
     private var linkMode: LinkMode? = null
     private var googlePaySupported: Boolean = false
     private var currency: String? = null
 
-    private val linkEnabled: Boolean
-        get() = linkMode != null
```

Comment:
Even if `linkMode` is not null, the merchant could disable Link due to various things (BDCC, CBF, LinkConfiguration`.